tests/testthat/test-board_folder.R

```diff
@@ -43,6 +44,27 @@ test_that("can deparse", {
   )
 })
 
+test_that("contents of manifest match", {
+  b <- board_folder(withr::local_tempfile())
+  pin_write(b, mtcars, "mtcars-csv", type = "csv")
+  pin_write(b, mtcars, "mtcars-json", type = "json")
+  write_board_manifest(b)
+
+  # names are correct
+  manifest <- yaml::read_yaml(fs::path(b$path, manifest_pin_yaml_filename))
+  expect_identical(names(manifest), pin_list(b))
+
+  # values (relative paths to versions) are correct
+  imap(
+    manifest,
+    ~ expect_identical(
```

Comment:
Is there some way you could construct this expected values so the structure is obvious at a glance? And maybe worthwhile to create a pin with more than one version?

Reply:
Added another version to one of the test pins 👍 

I thought a while about a way to construct the expected values, but it is not obvious to me with the versions. We currently handle that with `is_testing()` in `version_name()` but that has caused problems as outlined in #640. I wonder if maybe the way to go is snapshot tests with a function to redact versions. I'll add that in the existing issue.